.babelrc

```diff
@@ -2,7 +2,7 @@
   "presets": [
     ["env", {
       "targets": {
-        "node": "4.5"
+        "node": 6
```

Comment:
So I changed it to exclude `transform-regenerator` instead, this way we won't need to modify tests.